streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -4318,7 +4318,7 @@ object ZStream extends ZStreamPlatformSpecificConstructors {
           bufArray  <- ZIO.succeed(Array.ofDim[Byte](chunkSize))
           bytesRead <- ZIO.attemptBlockingIO(is.read(bufArray)).asSomeError
           bytes <- if (bytesRead < 0)
-                     ZIO.fail(None)
+                     Exit.failNone
```

Comment:
Maybe better done as a separate PR, but maybe we should store `Exit.succeed(Chunk.empty)` in a val and use it below here

Reply:
Hehe: https://github.com/zio/zio/pull/9360

Reply:
Wow 😆